### PR TITLE
Acrolinx Batch 1 Edits

### PR DESCRIPTION
**Pull request by daphnemamsft**

### CODE REVIEW

File: docs-ref-conceptual/azure-cli-vm-tutorial-2.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ ms.custom: devx-track-azurecli, seo-azure-cli
 keywords: azure cli create vnet, virtual network in azure cli, subnet in virtual network
 ---
 
-# 2 - Create a virtual network
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/azure-cli-vm-tutorial-2.md:15]: @daphnemamsft , Tutorial H1s should use the numer `2` not the word `two`.  See this Python tutorial for an example: https://learn.microsoft.com/en-us/azure/developer/python/tutorial-containerize-deploy-python-web-app-azure-02?tabs=sample-app-git-clone%2Cvscode-docker%2Cterminal-bash%2Cmongodb-local

File: docs-ref-conceptual/reference-types-and-status.md [hunk 2]
```diff
@@ -57,34 +57,34 @@ az extension list-available --output table
...
 
 ## What is reference status?
 
-Regardless of reference type, Azure CLI references fall into three status categories: **GA** (Generally Available), **public preview** or **experimental**. It is the reference command status, not type, that determines stability and support level.
+Regardless of reference type, Azure CLI references fall into three status categories: **GA** (Generally Available), **public preview** or **experimental**. It's the reference command status (not type) that determines stability and support level.
 
 | | GA  | Public preview | Experimental
 |-|-|-|-|
-| **Stability** | Permanent | Can change in response to customer feedback. Is subject to the terms of [Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/). | Can change in response to customer feedback. Will often migrate to public preview.  Can be removed.
+| **Stability** | Permanent | Can change in response to customer feedback. Is subject to the terms of [Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/). | Can change in response to customer feedback. Can migrate to public preview.  Can be removed.
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/reference-types-and-status.md:81]: @daphnemamsft , If Acrolinx doesn't like `will often` we need to find a different word that indicates time.  Try `Often migrates`

**Review by dbradish-microsoft — CHANGES_REQUESTED** [review]

File: docs-ref-conceptual/reference-types-and-status.md (hunk 2 above)
Comment (daphnemamsft) [reply docs-ref-conceptual/reference-types-and-status.md]: Ok, I have changed the wording

File: docs-ref-conceptual/azure-cli-vm-tutorial-2.md (hunk 1 above)
Comment (daphnemamsft) [reply docs-ref-conceptual/azure-cli-vm-tutorial-2.md:15]: Sounds good, I will go ahead and revise this and take this into account for the other docs

**Review by dbradish-microsoft — APPROVED** [review]

**Pull request merged**